site/docs/styles.css

```diff
@@ -38,7 +40,7 @@
 
 .vocs_CodeGroup {
   border-radius: 8px;
-  overflow: hidden;
+  overflow: visible;
```

Comment:
allow twoslash popups to overflow code boxes

Reply:
niiice

Reply:
Hopefullly this will not cause a regression with Mobile Web, but if it does we need to re-think this.